app/views/layouts/application.html.erb

```diff
@@ -23,6 +23,7 @@
   </nav>
 
   <body>
+    <script src="https://prod.flat-cdn.com/embed-js/v1.3.0/embed.min.js"></script>
```

Comment:
We may want to bump this down a few lines in the file. There are comparable file loads for bootstrap which it recommended loading at the end of the page - see my open PR. That also seems consistent with what Brian indicated.